pkg/kubelet/util/util.go

```diff
@@ -60,3 +66,47 @@ func GetContainerByIndex(containers []v1.Container, statuses []v1.ContainerStatu
 	}
 	return containers[idx], true
 }
+
+var (
+	tmpfsNoswapOptionSupported        bool
+	tmpfsNoswapOptionAvailabilityOnce sync.Once
+)
+
+func IsTmpfsNoswapOptionSupported(mounter mount.Interface) bool {
+	isTmpfsNoswapOptionSupportedHelper := func() bool {
+		if sysruntime.GOOS == "windows" {
+			return false
+		}
+
+		mountDir := filepath.Join("/", "tmp", "tmpfs-noswap-test")
```

Comment:
os.MkdirTemp would probably be a better fit here.

Reply:
Done, thanks!